.github/auto_assign.yml

```diff
@@ -6,16 +6,10 @@ addAssignees: false
 
 # A list of reviewers to be added to pull requests (GitHub user name)
 reviewers:
-  - darrenj
```

Comment:
We will probably need to fix these once it goes in. We shouldn't have updated this in the feature branch.